src/HelixServer.js

```diff
@@ -90,12 +90,19 @@ async function executeTemplate(ctx) {
     REPO_RAW_ROOT: `${ctx.strain.content.rawRoot}/`, // the pipeline needs the final slash here
     REPO_API_ROOT: `${ctx.strain.content.apiRoot}/`,
   };
+
   if (ctx.body) {
     // add post params to action params
     Object.keys(ctx.body).forEach((key) => {
       actionParams[key] = ctx.body[key];
     });
   }
+  if (ctx._cfg._params) {
+    // add argument action params
+    Object.keys(ctx._cfg._params).forEach((key) => {
+      actionParams[key.toString()] = ctx._cfg._params[key];
```

Comment:
`key.toString()` is not needed. as all keys are strings, I think.

Reply:
maybe add a _convenience_ getter to `RequestContext`: `get actionParams()`, so that you don't need to access the _internal_ field `_cfg` here.